geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/RedisStatsIntegrationTest.java

```diff
@@ -468,33 +479,33 @@ public void NetworkKiloBytesReadOverLastSecond_shouldReturnCorrectData() {
 
   }
 
-
   // ######################### Clients Section #################################
 
   @Test
   public void clientsStat_withConnectAndClose_isCorrect() {
 
-    jedis = new Jedis("localhost", server.getPort(), TIMEOUT);
-    jedis.ping();
+    Jedis jedis2 = new Jedis("localhost", server.getPort(), TIMEOUT);
+    jedis2.ping();
 
-    assertThat(redisStats.getConnectedClients()).isEqualTo(1);
+    assertThat(redisStats.getConnectedClients()).isEqualTo(preTestConnectedClients + 1);
 
-    jedis.close();
+    jedis2.close();
     GeodeAwaitility.await().atMost(Duration.ofSeconds(2))
-        .untilAsserted(() -> assertThat(redisStats.getConnectedClients()).isEqualTo(0));
+        .untilAsserted(
+            () -> assertThat(redisStats.getConnectedClients()).isEqualTo(preTestConnectedClients));
   }
 
   @Test
   public void connectionsReceivedStat_shouldIncrement_WhenNewConnectionOccurs() {
 
-    jedis = new Jedis("localhost", server.getPort(), TIMEOUT);
-    jedis.ping();
+    Jedis jedis2 = new Jedis("localhost", server.getPort(), TIMEOUT);
+    jedis2.ping();
 
-    assertThat(redisStats.getConnectionsReceived()).isEqualTo(1);
+    assertThat(redisStats.getConnectionsReceived()).isEqualTo(preTestConnectionsReceived + 1);
 
-    jedis.close();
+    jedis2.close();
 
-    assertThat(redisStats.getConnectionsReceived()).isEqualTo(1);
+    assertThat(redisStats.getConnectionsReceived()).isEqualTo(preTestConnectionsReceived + 1);
```

Comment:
I don't think this assertion is correct. It should be doing what the previous test is doing:
```
GeodeAwaitility.await().atMost(Duration.ofSeconds(2))
        .untilAsserted(() -> assertThat(redisStats.getConnectedClients()).isEqualTo(preTestConnectedClients));
```

Reply:
I'm not sure how we can make this more clear, but I see how this is confusing-  this test is testing the connectionsReceived parameter, which keeps the history of connections received, even after they have been closed.   (the getConnectedlClient param, in the test above is about current active connections only)

Reply:
OK - got it. That does make sense. I think the test method name does make it clearer, but I wasn't paying close attention to that :/